index.html

```diff
@@ -1196,6 +1280,12 @@ <h3>Identifiers</h3>
           </li>
         </ul>
 
+        <p class="note">
+If a credential contains a credentialSubject property and it, in turns 
+contains an id property, the value of the id property, if present, is to be interpreted 
+as an identifier for the corresponding credential subject.
+        </p>
```

Comment:
```suggestion
        
```
What does this section have to do with "intended audience, purpose, goals" etc.  I suggest this paragraph

Reply:
> What does this section have to do with "intended audience, purpose, goals" etc. I suggest this paragraph gets pulled, and that where possible we look at one topic per PR

Killing 2 birds with one stone @mprorock per https://github.com/w3c/vc-data-model/pull/800#issuecomment-916302702

Reply:
It's generally not considered friendly collaboration to respond to a request with a "No" and a "conversation resolved" button click.  Both parties generally must agree that the conversation has been resolved, before that button should be clicked.